packages/util/src/withdrawal.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Same thing here, could disabling no-use-before-define be avoided?

Reply:
:heavy_check_mark: